src/main/java/com/fasterxml/jackson/databind/deser/std/UntypedObjectDeserializer.java

```diff
@@ -27,8 +30,8 @@
  */
 @JacksonStdImpl
 public class UntypedObjectDeserializer
-    extends StdDeserializer<Object>
-    implements ResolvableDeserializer, ContextualDeserializer
+        extends StdDeserializer<Object>
```

Comment:
Whitespace changes make this diff rather hard to read :-(

Reply:
Yes sorry about that. Should I fix that and make another pr?

Reply:
@Niken-physics if you fix it in this PR, the diff will update and it should be fine.